### ISSUE TITLE
Upgrade HTTP to HTTPS option for web

### ISSUE THREAD

**Issue by waspennator** [ruffle-rs/ruffle#11320]
**Describe the feature**
Now that the desktop has an option for working around swfs made during the era of sites using http, it would be cool if the web versions of ruffle could also get a similar option for it, least for the game's and animations that have been abandoned with old http era sitelocks like #5989

**Comment by n0samu** [comment]
The title of this issue incorrectly describes what is actually being asked for. Really what's being asked for is an https => http compatibility rule for sitelocked Flash content that checks specifically for an HTTP URL. An example of content with this type of faulty sitelock is #9967.

**Comment by waspennator** [comment]
> The title of this issue incorrectly describes what is actually being asked for. Really what's being asked for is an https => http compatibility rule for sitelocked Flash content that checks specifically for an HTTP URL. An example of content with this type of faulty sitelock is #9967.

Yeah, that's honestly a better way of putting it, moreso cause there are certain games or animations that are effected by this, kinda hard to work around the http stuff on ruffle on an S22 compared to a windows laptop.
